ci/pipeline.yml

```diff
@@ -8,6 +8,42 @@ jobs:
     params:
       build: snort-release-source/ci/docker/pulledpork
 
+- name: update-release-jammy-git
```

Comment:
Not your problem for this PR I guess, but we should really. merge the jammy branch to `main` now that everything is running on Jammy so we can avoid this kind of duplication